Stratis.Bitcoin.Features.Consensus/Checkpoints.cs

```diff
@@ -138,6 +138,7 @@ public Checkpoints(Network network)
         {
             if (network.Equals(Network.Main)) this.checkpoints = bitcoinMainnetCheckpoints;
             else if (network.Equals(Network.TestNet)) this.checkpoints = bitcoinTestnetCheckpoints;
+            else if (network.Equals(Network.RegTest)) this.checkpoints = new Dictionary<int, CheckpointInfo>();
```

Comment:
I don't get it. I'm reading this over and over again and I don't get it :(
What am I missing? Why is this needed?

Reply:
This is only for the truly knowledgeable in blockchain ;)

Reply:
Seriously, this is related to an issue in NBitcoin where POS sets static flags when initiating the stratis network.
I am working on fixing this.

Reply:
I still don't understand this. Do you say that network.Equals gives TRUE for some of the IFs we've had there before in case the network was RegTest and not to go with the final else?

Reply:
OK, you've explained this to me. Weird!